src/Gridify/GridifyExtensions.cs

```diff
@@ -483,4 +485,243 @@ public static Paging<T> Gridify<T>(this IQueryable<T> query, Action<IGridifyQuer
    }
 
    #endregion
+
+   /// <summary>
+   /// Validates the GridifyQuery including field names and value type compatibility.
+   /// </summary>
+   /// <param name="filtering">The filtering query to validate.</param>
+   /// <param name="validationErrors">List of validation error messages if validation fails.</param>
+   /// <param name="mapper">Optional custom mapper.</param>
+   /// <returns>True if the query is valid; otherwise, false.</returns>
+   public static bool IsValid<T>(
```

Comment:
This is actually a great idea but we need to have a consistent behavior between these two overloads, right now this overload complains about some mismatched value types but the other one return true

I would suggest to call this overload in the old one, and discard the `out` parameter `_`.